src/utils/priceFetch.ts

```diff
@@ -1,22 +1,34 @@
 import axios from 'axios';
 import { getSentryTracker } from '../init';
 
+const FETCH_WINDOW_IN_MS = 60 * 1000;
+const ATTEMPTS = 5;
+
+let price = 0;
+let timestampInMS = 0;
+
```

Comment:
Add tests for this function. Seems good candidate to test.

Reply:
Yes, let me do this today and I'll update the PR.